Model/Indexer/AsyncEventSubscriber.php

```diff
@@ -149,7 +160,7 @@ public function execute($ids)
     /**
      * @inheritDoc
      */
-    public function executeByDimensions(array $dimensions, Traversable $entityIds)
+    public function executeByDimensions(array $dimensions, Traversable $entityIds = null)
```

Comment:
This looks familiar. Same `?Traversable` type restriction we discussed last time?

Reply:
Oh yes.That's because v2 supports PHP 7.0. Nullable types were introduced in 7.1 :+1: